src/dfi/rpc_accounts.cpp

```diff
@@ -2117,77 +2117,96 @@ UniValue sendtokenstoaddress(const JSONRPCRequest &request) {
 
 UniValue transferdomain(const JSONRPCRequest &request) {
     auto pwallet = GetWallet(request);
-    // TODO: Add support for non-JSON parameteric input that's human friendly and intuitive
-    RPCHelpMan{
+
+    if (!request.params[0].isArray()) {
+        RPCHelpMan{
```

Comment:
Help params need to be outside of the if branch and with both help messages in the same unit. 
Only the final `Check` call inside.

Reply:
not sure i'm doing it correctly
refined https://github.com/DeFiCh/ain/pull/2679/commits/9e23c237d15170b0f7353851384b9600a1705456

Reply:
custom made RPCHelpMan https://github.com/DeFiCh/ain/pull/2679/commits/480b0806b7c27105a32ee0f0418c04efcabf8728
```
transferdomain
Creates (and submits to local node and network) a tx to transfer assets across domains. DVM to EVM/EVM to DVM, etc.

Arguments:
1. array | string                       (json array | string, required) A json array of src and dst json objects
    [
    {                        (json object)
        "src": {               (json object) Source arguments
        "address": "str",    (string, required) Source address
        "amount": "str",     (string, required) Amount transfered, the value is amount in amount@token format
        "domain": n,         (numeric, required) Domain of source: 2 - DVM, 3 - EVM
        },
        "dst": {               (json object) Destination arguments
        "address": "str",    (string, required) Destination address
        "amount": "str",     (string, required) Amount transfered, the value is amount in amount@token format
        "domain": n,         (numeric, required) Domain of source: 2 - DVM, 3 - EVM
        },
        "nonce": n,            (numeric) Optional parameter to specify the transaction nonce
    },
    ...
    ]
   from                         (string, required) the source address of sender
2. to                           (string, required) the destination address of sender
3. tokenAmount                  (string, required) in amount@token format
4. nonce

Result:
"hash"                  (string) The hex-encoded hash of broadcasted transaction

Examples:
> defi-cli transferdomain '[{"src":{"address":"<DFI_address>", "amount":"1.0@DFI", "domain": 2}, "dst":{"address":"<ETH_address>", "amount":"1.0@DFI", "domain": 3}}]'
> defi-cli transferdomain '[{"src":{"address":"<ETH_address>", "amount":"1.0@DFI", "domain": 3}, "dst":{"address":"<DFI_address>", "amount":"1.0@DFI", "domain": 2}}]'
> defi-cli transferdomain "from" "to" "100@DFI"
> defi-cli transferdomain "from", "to", 100@BTC 2"
```